AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -65,6 +67,17 @@
  */
 @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
 
+/**
+ * @abstract Return the calculated layout.
+ *
+ * @discussion Ideal for use by subclasses in -layout, having already prompted their subnodes to calculate their size by
```

Comment:
As of this patch, is it still necessary for each node to call -measure: on the subnodes in -calculateLayoutThatFits:?  That is surely necessary for manual layout areas, but for layout spec areas, the -measure: calls will be prompted automatically — right?

Update the comment to be a bit more explicit about this (feel free to add another sentence describing the automatic layout / spec case and clarify the first is for manual / nodes that override -calculateLayoutThatFits: